app-admin/pinkybar/pinkybar-9999.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/su8/pinky-bar.git"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit git-r3 python-single-r1 autotools
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://github.com/su8/pinky-bar"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+alsa colours cpp drivetemp drivetemp-light dvd libnl lisp lua mpd ncurses
+	+net +pci perl python2 r ruby sensors slang smartemp tcl weather x11"
+
+DEPEND="
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
+	sys-apps/gawk
+	sys-devel/m4
+	dev-lang/perl
+"
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	drivetemp? ( net-misc/curl app-admin/hddtemp )
+	drivetemp-light? ( app-admin/hddtemp )
+	dvd? ( dev-libs/libcdio )
+	libnl? ( >=dev-libs/libnl-3.2.27 virtual/pkgconfig )
+	lisp? ( dev-lisp/ecls )
+	lua? ( dev-lang/lua )
+	mpd? ( media-sound/mpd media-libs/libmpdclient )
+	ncurses? ( sys-libs/ncurses )
+	net? ( sys-apps/iproute2 )
+	pci? ( sys-apps/pciutils )
+	python2? ( ${PYTHON_DEPS} )
+	r? ( dev-lang/R )
+	ruby? ( dev-lang/ruby )
+	sensors? ( sys-apps/lm_sensors )
+	slang? ( sys-libs/slang )
+	smartemp? ( sys-apps/smartmontools )
+	tcl? ( dev-lang/tcl )
+	weather? ( net-misc/curl app-arch/gzip )
+	x11? ( x11-libs/libX11 )
+
+"
+REQUIRED_USE="
+	drivetemp? ( !smartemp )
+	drivetemp-light? ( !smartemp )
+	ncurses? ( !x11 )
+	smartemp? ( !drivetemp !drivetemp-light )
+	x11? ( !ncurses )
+"
+
+src_prepare() {
+	default
+
+	elog 'Generating Makefiles'
```

Comment:
Is this something user needs to know long after the package build is complete? I doubt it, so `einfo` is valid here.